### PR TITLE
Add Traefik to Adopters

### PR DESCRIPTION
**Pull request by anvega**
Add Traefik to Adopters

### CODE REVIEW

**Comment by faisal-memon** [comment]
extra white space in there somewhere:
```
ADOPTERS.md:62:48 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
```

File: ADOPTERS.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ SPIFFE and SPIRE have integrations available with a number of open-source projec
 * [Sigstore](https://github.com/sigstore/fulcio)  
 * [Tekton](https://github.com/tektoncd/chains)  
 * [Tornjak](https://github.com/spiffe/tornjak)  
+* [Traefik](https://github.com/traefik/traefik)
```
Comment (faisal-memon) [inline ADOPTERS.md:62]:
```suggestion
* [Traefik](https://github.com/traefik/traefik)
```

**Comment by azdagron** [comment]
@anvega would you mind fixing up DCO signoff on your commit and fixing up the whitespace? I can fix the whitespace, but not the signoff :)

**Comment by evan2645** [comment]
Hey @jlevesy - thank you for adding SPIFFE support to Traefik!! ❤️ 

We'd love to add Traefik to our adopters list, but we need a DCO. Any chance you're up for sending us a quick PR like this one w/ a DCO attached? 🙌

**Comment by jlevesy** [comment]
Sounds good!

But shouldn't this come from actual Traefik maintainers? 

I'm only a (very) occasional contributor ^^

Ping @juliens for approval :)

**Pull request closed without merging**